trin-core/src/portalnet/overlay_service.rs

```diff
@@ -975,7 +975,7 @@ where
             Ok(Some(content)) => {
                 // Check content size and initiate uTP connection if the size is over the threshold
                 // TODO: Properly calculate max content size
-                if content.len() < 1000 {
+                if content.len() <= 1028 {
```

Comment:
This increase in size was required so that the `eth_getBlockByHash` scenario in `ethportal-peertest` doesn't trigger a utp transfer. It seemed like an alright change imo, since it's only marginally larger, and we still have the todo to properly calculate max content size

Reply:
Always nice to link the specific issue when mentioning it in the PR :)

Reply:
https://github.com/ethereum/trin/issues/523